app-server/app/src/main/resources/application.conf

```diff
@@ -53,4 +53,7 @@ auth0 {
   # Development Secret Only
   secret = ""
   secret = ${?AUTH0_CLIENT_SECRET}
+  # Auth0 Bearer with proper permissions for Management API
+  bearer = ""
+  bearer = ${?AUTH0_MANAGEMENT_BEARER}
```

Comment:
Another potential point of failure is that if we update this token to not have all the permissions the code needs, there may be runtime failures. I wonder if we can make a test of some sort that decodes the token (it is simple JWT) and ensures that the right permissions are enabled:

```json
{
  "aud": "UlV...",
  "scopes": {
    "device_credentials": {
      "actions": [
        "read",
        "create",
        "delete",
        "update"
      ]
    }
  },
  "iat": 147000000,
  "jti": "dd1..."
}
```